{"uuid":"3af55e49-2e5b-4bdf-95e7-a4a0ab6f6ef9","historyId":"d3eb9dd6c8bc228ecae8403629e9ac11","fullName":"tests.BaseTest.setup","labels":[{"name":"package","value":"tests.app_functional_tests.tests.TestForHomePage"},{"name":"testClass","value":"tests.app_functional_tests.tests.TestForHomePage"},{"name":"testMethod","value":"setup"},{"name":"parentSuite","value":"Parallel Test Suite"},{"name":"suite","value":"Device1"},{"name":"subSuite","value":"tests.app_functional_tests.tests.TestForHomePage"},{"name":"host","value":"Sedas-MacBook-Pro.local"},{"name":"thread","value":"3638@Sedas-MacBook-Pro.local.TestNG-tests-1(32)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"AS_ID","value":"-1"}],"links":[],"name":"setup","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: Device emulator-5554 was not in the list of connected devices \nHost info: host: 'Sedas-MacBook-Pro.local', ip: 'fe80:0:0:0:141e:a485:4991:fa24%en0'\nBuild info: version: '4.29.0', revision: '5fc1ec94cb'\nSystem info: os.name: 'Mac OS X', os.arch: 'aarch64', os.version: '15.3.1', java.version: '23.0.2'\nDriver info: io.appium.java_client.android.AndroidDriver\nCommand: [null, newSession {capabilities=[Capabilities {appium:app: /Users/norayrantonyan/Deskt..., appium:appActivity: org.wikipedia.main.MainActi..., appium:appPackage: org.wikipedia, appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, appium:udid: emulator-5554, platformName: ANDROID}]}]\nCapabilities {appium:app: /Users/norayrantonyan/Deskt..., appium:appActivity: org.wikipedia.main.MainActi..., appium:appPackage: org.wikipedia, appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, appium:udid: emulator-5554, platformName: ANDROID}","trace":"org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: Device emulator-5554 was not in the list of connected devices \nHost info: host: 'Sedas-MacBook-Pro.local', ip: 'fe80:0:0:0:141e:a485:4991:fa24%en0'\nBuild info: version: '4.29.0', revision: '5fc1ec94cb'\nSystem info: os.name: 'Mac OS X', os.arch: 'aarch64', os.version: '15.3.1', java.version: '23.0.2'\nDriver info: io.appium.java_client.android.AndroidDriver\nCommand: [null, newSession {capabilities=[Capabilities {appium:app: /Users/norayrantonyan/Deskt..., appium:appActivity: org.wikipedia.main.MainActi..., appium:appPackage: org.wikipedia, appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, appium:udid: emulator-5554, platformName: ANDROID}]}]\nCapabilities {appium:app: /Users/norayrantonyan/Deskt..., appium:appActivity: org.wikipedia.main.MainActi..., appium:appPackage: org.wikipedia, appium:automationName: UiAutomator2, appium:deviceName: emulator-5554, appium:udid: emulator-5554, platformName: ANDROID}\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:114)\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:75)\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:61)\n\tat io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:178)\n\tat io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:243)\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)\n\tat io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:313)\n\tat org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:174)\n\tat io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:101)\n\tat io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:113)\n\tat io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:109)\n\tat driver_manager.DriverManager.getDriverInstance(DriverManager.java:28)\n\tat tests.BaseTest.setup(BaseTest.java:29)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\n\tat org.testng.TestRunner.run(TestRunner.java:616)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)\n\tat org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)\n\tat java.base/java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:317)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java)\n\tat --- Async.Stack.Trace --- (captured by IntelliJ IDEA debugger)\n\tat java.base/java.util.concurrent.FutureTask.<init>(FutureTask.java:132)\n\tat java.base/java.util.concurrent.AbstractExecutorService.newTaskFor(AbstractExecutorService.java:113)\n\tat java.base/java.util.concurrent.AbstractExecutorService.invokeAll(AbstractExecutorService.java:305)\n\tat org.testng.internal.thread.ThreadUtil.execute(ThreadUtil.java:70)\n\tat org.testng.SuiteRunner.runInParallelTestMode(SuiteRunner.java:442)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:371)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"udid","value":"emulator-5554"},{"name":"port","value":"4723"}],"start":1741009699032,"stop":1741009699033}